pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -230,6 +230,9 @@ type LogCollectorSpec struct {
 	// Periodicity is the periodicity of the log rotation
 	// +optional
 	Periodicity string `json:"periodicity,omitempty"`
+	// MaxLogSize is the maximum size of the log per ceph daemons. For example: 500M, 1G
+	// +optional
+	MaxLogSize string `json:"maxLogSize"`
```

Comment:
```suggestion
	MaxLogSize string `json:"maxLogSize,omitempty"`
```

Reply:
Should this be a resources.Quantity instead of a string? We can avoid extra bashy manipulation with this.

Reply:
PTAL now and let me know we using `resources.Quantity`  will help further

Reply:
Please do change it to `resources.Quantity`. Then the kubebuilder:validation tag should be added automatically instead of you adding it explicitly
```suggestion
	MaxLogSize resources.Quantity `json:"maxLogSize"`
```